duckduckgo/macos-browser
# Design adjustments for Network Protection onboarding views

Task/Issue URL: 

## Description

Addresses design adjustments requested for the new Network Protection onboarding views.

Work done:
- Update to the latest design changes
- Update to the latest assets
- Update to the latest copy
- The Allow-VPN step will now only trigger if the user disallows creating the VPN configuration.

## Scope:

Address design feedback.  Nothing outside of design feedback is in scope for this PR.

## Testing:

This PR will be accompanied by a design-review build, so I'd encourage the technical reviewer to focus on the logical changes mainly (and not so much on coloring, fonts, spacing, etc).

1. Restart Network Protection to start fresh.
2. Start NetP, the allow-system-extension onboarding step view should be shown in the popover.
3. After allowing the extension, allow-vpn-configuration onboarding step view should **NOT** be shown.
4. Do not allow the app to create the VPN configuration.
5. The allow-vpn-configuration onboarding step view should be shown in the NetP popover.
6. Follow the instructions and ensure the onboarding view goes away once the setup is complete.

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
